docs/output/targeted.md

```diff
@@ -0,0 +1,238 @@
+---
+order: 1
+---
+
+# nf-core/crisprseq: Output
+
+## Introduction
+
+This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+
+The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
+
+## Pipeline overview
+
+The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
+
+- [Preprocessing](#preprocessing)
+  - [Sequences](#sequences) - Input sequence preparation (reference, protospacer, template)
+  - [cat](#cat) - Concatenate sample fastq files if required
+  - [Pear](#pear) - Join double-end reads if required
+  - [FastQC](#fastqc) - Read Quality Control
+  - [Adapters](#adapters) - Find adapters (Overrepresented sequences) in reads
+  - [Cutadapt](#cutadapt) - Trim adapters
+  - [Seqtk](#seqtk) - Mask low-quality bases
+  <!-- -UMI(#umi) -->
+- [Mapping](#mapping)
+  - [minimap2](#minimap2) - Mapping reads to reference
+  - [BWA](#bwa) - Mapping reads to reference
+  - [bowtie2](#bowtie2) - Mapping reads to reference
+- [Edits calling](#edits-calling)
+  - [CIGAR](#cigar) - Parse CIGAR to call edits
+- [MultiQC](#multiqc) - Aggregate report describing results and QC from the whole pipeline
+- [Pipeline information](#pipeline-information) - Report metrics generated during the workflow execution
+
+## Preprocessing
+
+### Sequences
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `preprocessing/sequences/`
+  - `*_reference.fasta`: Sequence used as a reference.
+  - `*_template.fasta`: Provided template sequence.
+  - `*_correctOrient.fasta`: Reference sequence in the correct orientation.
+  - `_NewReference.fasta`: New reference generated from adding the changes made by the template to the original reference.
+  - `*_template-align.bam`: Alignment of the new reference (with template changes) to the original reference.
+
+</details>
+
+Contains the input sequences (reference, protospacer and template). Sequences are preprocessed as required:
+
+- The reference is returned in the correct orientation.
+
+  :::info
```

Comment:
This is not formatted correctly. It really shows the `:::` in the md file. 
```suggestion
  > **_NOTE:_**
```

Reply:
new web